nss/src/group/mod.rs

```diff
@@ -1,33 +1,33 @@
 use crate::{info, REQUEST_TIMEOUT};
 use libc::gid_t;
-use libnss::group::{Group, GroupHooks};
+use libnss::group::{Group as NSSGroup, GroupHooks};
```

Comment:
We are in the NSS context here, so we should go the other way around: use the type alias for the `authd` type instead of the NSS type.